application/helpers/common_helper.php

```diff
@@ -1215,7 +1215,7 @@ function createCompleteSGQA($iSurveyID, $aFilters, $sLanguage)
 
                 //go through all the (multiple) answers
                 foreach ($result as $row) {
-                    $myfield2 = $myfield . reset($row);
+                    $myfield2 = $myfield . $row['title'];
```

Comment:
Gloups ? 
Why it don't break for GUI statistics here ? Not used for GUI ?

Reply:
That method is only used on RC.
Seems there is a bit of code duplication and the GUI part was working fine :)